lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +198,89 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(full_url, article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        article = article_soup.find_all('div', {'itemprop': 'articleBody'})[0]
+        article_list = article.find_all('p')
+        paragraphs = [par.text for par in article_list]
+        self.article.text = '\n'.join(paragraphs)
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        article = article_soup.find_all('div', {'itemprop': 'headline'})[0]
```

Comment:
if you are sure there is only one element, use find instead of find_all

Reply:
Done